.github/workflows/ci.yaml

```diff
@@ -18,30 +23,38 @@ jobs:
     runs-on: ${{ matrix.OS }}-latest
     name: "tests"
     strategy:
+      fail-fast: false
       matrix:
-        PYTHON:
-          - 3.8
-          - 3.9
-        OS:
-          - ubuntu
-          - windows
-          - macos
+        python-version: [3.8, 3.9]
+        os: ['ubuntu', 'windows', 'macos']
```

Comment:
I would prefer at least the Python versions the other way. I think bullet-style lists are more common in YAML than JSON-style, and they make for cleaner diffs. Supported Python versions will change with some regularity.

OS versions might change if we need to support something other than `latest`, but for as long as we only need `latest`, this gives nicer reporting in the actions overview. Actually, might be better to put `os` before `python-version`: "(ubuntu, 3.9)" reads nicer than "(3.9, ubuntu)"